conf/resouces/stubs-json-payloads/income/LAA/C4/SA-single-tax-year-single-return-list.json

```diff
@@ -0,0 +1,25 @@
+{
+  "summary": "Self Assessment income details single tax year single return list LAA C4",
+  "value": {
+    "sa": [
+      {
+        "taxYear": "2019",
+        "returnList": [
+          {
+            "busStartDate": "2018-02-28",
```

Comment:
busStartDate and busEndDate not needed for C4 This will need to be actioned on all LAA C4 SA payloads